packages/framework/src/Component/Utils/Utils.php

```diff
@@ -16,6 +16,19 @@ public static function ifNull(mixed $testVariable, mixed $default): mixed
         return $testVariable ?? $default;
     }
 
+    /**
+     * @param string|null $string
+     * @return string|null
+     */
+    public static function emptyStringToNull(?string $string): ?string
```

Comment:
I believe the same feature is already provided by `\Shopsys\FrameworkBundle\Component\String\TransformStringHelper::getTrimmedStringOrNullOnEmpty()`.

when even the returned string is trimmed (which I think should be)